### ISSUE TITLE
Custom Model with 1 Class 54 MB file size

### ISSUE THREAD

**Issue by lgeuder** [ultralytics/yolov5#2279]
## ❔Question
Why is my 'best.pt' file 54 MB large after training on Yolov5s? 
Compared with the coco 5s model which is about 14MB large I am starting to feel like I did something wrong.
## Additional context
thats what i ran:
`python train.py  --cfg './models/yolov5s.yaml'  --batch-size 64 --data 'data.yaml' `

**Comment by glenn-jocher** [comment]
@lgeuder after training completes you will have an FP16 model. 

During training checkpoints contain FP16 models and FP32 optimizer gradients.

**Comment by lgeuder** [comment]
@glenn-jocher I might have misunderstood the workflow of training data. If i run the detection succesfully, do i have to run further trainings again?

**Comment by glenn-jocher** [comment]
@lgeuder I'm not sure what you mean. You don't have to do anything. Standard workflow is to 1) train a model, and 2) use it. During training checkpoints are loaded with various extra attributes that are required for resuming trainings. When a training completes these are stripped and the checkpoint sizes are then the same as the official models.

**Comment by lgeuder** [comment]
@glenn-jocher my training process is finished but the the best.pt file i got in /runs/../weights is 54 mb. Should i run it again? Do you recommend any customizations due to the fact that i only need 2 classes. My usecase is pretty much like the blood cells one. I am not aiming to get high precision detections all I need is FPS. The target is running stable with more than 1 fps on a raspberry pie 😅

**Comment by glenn-jocher** [comment]
@lgeuder if your file is larger than 14 MB it is likely training was interrupted or you are training with an older codebase. You can manually strip optimizers from a checkpoint if you want using the same method as train.py:
https://github.com/ultralytics/yolov5/blob/0070995bd58629d4628d11b1c8de9788aa55379b/train.py#L397-L401

**Comment by lgeuder** [comment]
okay thanks already, i‘ll try
